app/src/main.cpp

```diff
@@ -97,10 +107,20 @@ int main(const int count, const char** args) {
     if (result.has_error()) {
         exit(EXIT_FAILURE);
     }
-    auto chainSpec = result.value();
-    connection_manager = std::make_shared<network::PeerManager>(runner, chainSpec.getBootNodes(), stop_handler);
+    auto chain_spec = std::make_shared<plc::core::chain::Spec>(result.value());
+    connection_manager = std::make_shared<network::PeerManager>(runner, chain_spec->getBootNodes(), stop_handler);
     stop_handler->add(connection_manager);
 
+    auto service = std::make_shared<plc::core::runtime::Service>(chain_spec, connection_manager, runner);
+    service->loadGenesisRuntime();
+
+    bool finished = false;
+    // auto timer = runner->makePeriodicTimer(std::chrono::milliseconds(2000), [&finished, connection_manager, runner, service]() {
```

Comment:
Should the commented code be removed? Otherwise better add a comment why it's needed/useful